public/themes/pterodactyl/js/frontend/files/src/actions.js

```diff
@@ -316,6 +319,76 @@ class ActionsClass {
         });
     }
 
+    addToList(event) {
+      const parent = $(event.target).closest('tr');
+
+      const nameBlock = $(parent).find('td[data-identifier="name"]');
+      const delPath = decodeURIComponent(nameBlock.data('path'));
+      const delName = decodeURIComponent(nameBlock.data('name'));
+
+      var item = delPath + delName;
+
+      //Determine if we're removing or adding the item
+      if(files.indexOf(item) != -1) {
+        files.splice($.inArray(item, files), 1)
+        parent.removeClass('warning').delay(200);
+      } else {
+        files.push(item);
+        fileselements.push(parent);
+
+        parent.addClass('warning').delay(200);
+      }
+
+    }
+
+    deleteSelected() {
+      var nameBlock;
+
+      swal({
+          type: 'warning',
+          title: '',
+          text: 'Are you sure you want to delete <code>' +
```

Comment:
I think this would render better if each file being deleted was wrapped in `<code>` tags, rather than all wrapped together in a single tag.